creation/web_base/condor_platform_select.sh

```diff
@@ -72,13 +72,14 @@ findversion_debian() {
   dist_id_line=$(grep "DISTRIB_ID" /etc/lsb-release)
   dist_rel_line=$(grep "DISTRIB_RELEASE" /etc/lsb-release)
   if [[ ${dist_id_line} == *"Debian"* ]]; then
-    [[ ${dist_rel_line:16:3} = "11." ]] && condor_os='linux-debian10' && return
+    [[ ${dist_rel_line:16:3} = "12." ]] && condor_os='linux-debian11' && return
```

Comment:
Should this be: `condor_os='linux-debian12'` (instead of `linux-debian11`) since `${dist_rel_line:16:3} = "12."` ?

Reply:
this is on purpose because there is no condor tarball for 12